yeesheen.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Yeesheen
+---
+
+## What I do
+I manage the Knowledge Base and in-app education programs at Optimizely (an A/B testing and experimentation platform)!
+
```

Comment:
@Yeesheen I had to wean my coworkers off of the one word variant cuz it feels super-outdated to me. And now we're trying to move to Help Center content and away from Knowledge Base altogether...